api/clients.js

```diff
@@ -0,0 +1,16 @@
+import axios from "./axios-instance";
+
+const createClient = async (dataNewClient) => {
+	try {
+		const { name, phoneNumber: phone, carBrand, carModel, licensePlate, email } = dataNewClient;
```

Comment:
Lets rename phoneNumber into phone in sections/admin/clients/new-client.jsx in useState to prevent creating object reqBody. It`s better to pass dataNewClient and not doint any tranformig here

P.S. Its my bad that I named this useState phoneNumber, it should be named `phone` from the beginning

Reply:
I don't think it's bad to have naming "phoneNumber" and make a request's body here, because this place for doing all of Axios (request to backend) is needed to do, but if u want - okay.